src/lava/magma/core/model/py/ports.py

```diff
@@ -16,8 +16,8 @@
 class AbstractPyPort(AbstractPortImplementation):
     """Abstract class for Ports implemented in Python.
 
-    It provides an interface to send and receive messages sent via channels. The
-    Communicating Sequential Processes (CSP) paradigm is followed in this
+    It provides an interface to send and receive messages via channels. The
+    Communicating Sequential Processes (CSP) paradigm is the inspiration of this
```

Comment:
The difference between CSP Ports and PyPorts should be spelled out.

Reply:
Perhaps useful to explain the relation between Ports and PyPorts. Ports at the Proc level provide an interface to connect ports. PyPorts provide an implementation for message exchange in Python. That is they provide send, receveive interfaces for OutPorts and InPorts, etc.